scripts/copy-helpers.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+mkdir ./packages/field-plugin/dist/{react,vue3}
```

Comment:
:O oh nice, did not know this syntax works

Reply:
I knew about this, but this is my first time actually using it 😍